biodatasets/osiris/osiris.py

```diff
@@ -236,7 +236,7 @@ def _get_entities(self, row: dict) -> List[dict]:
                 "offsets": [variant["offsets"]],
                 "text": [variant["v_lex"]],
                 "type": "variant",
-                "normalized":  [{"db_name": "dbSNP" if variant["v_id"] == "No" else "NCBI Gene", "db_id": variant["v_norm"] if variant["v_id"] == "No" else variant["v_id"]}]
+                "normalized":  [{db_name": "HGVS-like" if variant["v_id"] == "No" else "dbSNP", "db_id": variant["v_norm"] if variant["v_id"] == "No" else variant["v_id"]}]
```

Comment:
There's one `"` missing :( Could you please fix this?

Reply:
Thank you very much for your comments! I fixed this bug😊